lib/runtime-core/src/vm.rs

```diff
@@ -547,15 +577,18 @@ pub struct LocalTable {
 unsafe impl Send for LocalTable {}
 
 impl LocalTable {
+    /// Offset base.
```

Comment:
As written this sounds like the base of an offset. What this is, is the offset into the structure, at which the field "base" lives. This is used in singlepass-generated code to look up where the "base" pointer is, given a pointer to a LocalTable. Suggestion, "offset to base"?

Reply:
I've updated this with your suggestion `Offset to base`.